hbase-server/src/main/java/org/apache/hadoop/hbase/namequeues/LogEventHandler.java

```diff
@@ -50,64 +50,83 @@
 import org.apache.hadoop.hbase.shaded.protobuf.generated.TooSlowLog.SlowLogPayload;
 
 /**
- * Event Handler run by disruptor ringbuffer consumer
+ * Event Handler run by disruptor ringbuffer consumer.
+ * Although this is generic implementation for namedQueue, it can have individual queue specific
```

Comment:
Why not follow strategy pattern, where individual queues specific logics would be left for the implementing strategy?

Reply:
Incorporated.